programs/clearing_house/src/controller/bank_balance.rs

```diff
@@ -63,10 +63,21 @@ pub fn update_bank_cumulative_interest(bank: &mut Bank, now: i64) -> ClearingHou
         utilization,
     } = calculate_accumulated_interest(bank, now)?;
 
-    if deposit_interest > 0 && borrow_interest > 1 {
+    // borrowers -> lenders IF fee here
+    let deposit_interest_for_stakers = deposit_interest
```

Comment:
don't love this, would rather  transform `deposit_interest_for_stakers` into a token amount and update a variable on bank e.g. `insurance_fund_fees`.

Two reasons:
1) Not sure how this pattern handles updating `bank.total_reserve_factor` and `bank.user_reserve_factor`?
2) If we want to send interest fees to multiple places (not just IF), this pattern won't support it

Reply:
can send hourly revenue to revenue vault (added todos in instruction)